src/components/CheckableButton/CheckableButton.scss

```diff
@@ -1,18 +1,18 @@
-@import '../../../../styles/common';
-@import '../../variables';
+@import '../../styles/common';
 
-$button-vertical-padding: (
-    resource-list(button-min-height) - line-height(body) - rem(2px)
-  ) / 2;
+$button-min-height: control-height();
```

Comment:
Why remove this from _variables?

Reply:
`_variables` Is scoped to the resource list & this variable was only used inside `CheckableButton`. Rather than adding scss to our API I just moved it to where it's used 😄